package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -303,20 +307,24 @@ class DiffusionMap(object):
 
     Methods
     -------
-    run()
+    run()line
         Constructs an anisotropic diffusion kernel and performs eigenvalue
         decomposition on it.
     transform(n_eigenvectors, time)
         Perform an embedding of a frame into the eigenvectors representing
         the collective coordinates.
+    
+    .. versionchanged: 2.2.0
+         :class: `DiffusionMap` now also accepts `AtomGroup`.
```

Comment:
typo - extra space between class and classname
```suggestion
         :class:`DiffusionMap` now also accepts `AtomGroup`.
```